app/src/main/java/com/sdp/swiftwallet/WalletActivity.java

```diff
@@ -4,16 +4,84 @@
 import androidx.constraintlayout.widget.ConstraintLayout;
 
 import android.annotation.SuppressLint;
+import android.app.Activity;
 import android.os.Bundle;
+import android.util.Log;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
 import android.widget.LinearLayout;
+import android.widget.ScrollView;
 import android.widget.TextView;
 
 import com.sdp.cryptowalletapp.R;
 
+import java.security.KeyStore;
+
 public class WalletActivity extends AppCompatActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_wallet);
+        createText();
+    }
+
+    private void createText() {
+        LinearLayout walletLayout = findViewById(R.id.wallet_layout2);
+
+        final int N = 20; // total number of linear layouts
+        final String[] labels = new String[]{"BitCoin", "Ethereum", "apple", "oranges","BitCoin", "Ethereum", "apple", "oranges","BitCoin", "Ethereum", "apple", "oranges","BitCoin", "Ethereum", "apple", "oranges","BitCoin", "Ethereum", "apple", "oranges"};
+        final double[] balances = {0, 0, 0, 1,0, 0, 0, 1,0, 0, 0, 1,0, 0, 0, 1,0, 0, 0, 1};
+        final LinearLayout[] currencyList = new LinearLayout[N];
+        LinearLayout listlayout = new LinearLayout(this);
+        listlayout.setOrientation(LinearLayout.VERTICAL);
+        listlayout.setShowDividers(LinearLayout.SHOW_DIVIDER_END);
+        listlayout.canScrollHorizontally(LinearLayout.HORIZONTAL);
+        listlayout.setPadding(100,0,0,0);
+
+        for (int i = 0; i < N; i++) {
+            // create layout for currency
+            LinearLayout currencylayout = new LinearLayout(this);
+            currencylayout.setOrientation(LinearLayout.HORIZONTAL);
+            currencylayout.setShowDividers(LinearLayout.SHOW_DIVIDER_MIDDLE);
```

Comment:
Wouldn't it be nicer to have some already defined Layouts and components instead of defining them with some constants right here ?